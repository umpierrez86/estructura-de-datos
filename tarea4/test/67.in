# implementar ordenadaPorModulo
ordenadaPorModulo 2
insertarAlInicio 50 50.0
ordenadaPorModulo 2
insertarAlInicio 50 60.0
ordenadaPorModulo 2
insertarAlInicio 11 11.0
ordenadaPorModulo 2
insertarAlFinal 40 40.0
ordenadaPorModulo 2
ordenadaPorModulo 3
ordenadaPorModulo 4
ordenadaPorModulo 10
ordenadaPorModulo 100
insertarAlFinal 54 0.0
insertarAlFinal 49 0.0
insertarAlFinal 121 0.0
insertarAlFinal 18 0.0
ordenadaPorModulo 9
ordenadaPorModulo 7
ordenadaPorModulo 11
ordenadaPorModulo 6
Fin