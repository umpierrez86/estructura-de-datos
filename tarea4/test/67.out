1>#  implementar ordenadaPorModulo.
2>
3>Se insertó (50,50.00) al inicio de la cadena.
4>(50,50.00)
5>Se insertó (50,60.00) al inicio de la cadena.
6>(50,60.00)(50,50.00)
7>Se insertó (11,11.00) al inicio de la cadena.
8>(50,60.00)(50,50.00)(11,11.00)
9>Se insertó (40,40.00) al final de la cadena.
10>(50,60.00)(50,50.00)(40,40.00)(11,11.00)
11>(40,40.00)(11,11.00)(50,60.00)(50,50.00)
12>(40,40.00)(50,60.00)(50,50.00)(11,11.00)
13>(50,60.00)(50,50.00)(40,40.00)(11,11.00)
14>(11,11.00)(40,40.00)(50,60.00)(50,50.00)
15>Se insertó (54,0.00) al final de la cadena.
16>Se insertó (49,0.00) al final de la cadena.
17>Se insertó (121,0.00) al final de la cadena.
18>Se insertó (18,0.00) al final de la cadena.
19>(54,0.00)(18,0.00)(11,11.00)(40,40.00)(49,0.00)(121,0.00)(50,60.00)(50,50.00)
20>(49,0.00)(50,60.00)(50,50.00)(121,0.00)(11,11.00)(18,0.00)(40,40.00)(54,0.00)
21>(11,11.00)(121,0.00)(49,0.00)(50,60.00)(50,50.00)(40,40.00)(18,0.00)(54,0.00)
22>(54,0.00)(18,0.00)(49,0.00)(121,0.00)(50,60.00)(50,50.00)(40,40.00)(11,11.00)
23>Fin.
